blueocean-web/.babelrc

```diff
@@ -0,0 +1,8 @@
+{
+    "presets": [
+        "es2015", "react", "stage-0"
+    ],
+    "plugins": [
+        "transform-decorators-legacy"
+    ]
+}
```

Comment:
My imagination, or did I see another `.babelrc` that looked the very same as this? Can we put this in the top level folder and not duplicate?